### ISSUE TITLE
Maximum call stack exceeded, uncaughtable

### ISSUE THREAD

**Issue by razenization** [telegraf/telegraf#1128]
## Context

* Telegraf.js Version: ^3.38.0
* Node.js Version: 14.3.0
* Operating System: Win10

## Current Behavior

Receiving the following exception: 
```
RangeError: Maximum call stack size exceeded
    at Object.object.<computed> [as isNative] (/usr/lib/node_modules/pm2/node_modules/source-map-support/source-map-support.js:332:55)
    at Object.CallSiteToString [as toString] (/usr/lib/node_modules/pm2/node_modules/source-map-support/source-map-support.js:265:12)
    at Function.prepareStackTrace (/usr/lib/node_modules/pm2/node_modules/source-map-support/source-map-support.js:416:37)
    at maybeOverridePrepareStackTrace (internal/errors.js:86:29)
    at prepareStackTrace (internal/errors.js:65:5)
    at formatError (internal/util/inspect.js:1085:19)
    at formatRaw (internal/util/inspect.js:892:14)
    at formatValue (internal/util/inspect.js:746:10)
    at inspect (internal/util/inspect.js:287:10)
    at formatWithOptions (internal/util/inspect.js:1910:40)
    at Object.Console.<computed> (internal/console/constructor.js:304:10)
    at Object.warn (internal/console/constructor.js:314:61)
    at Object.consoleCall (<anonymous>)
    at execute (/var/www/html/ponchesme/backend/node_modules/telegraf/composer.js:374:12)
    at /var/www/html/ponchesme/backend/node_modules/telegraf/composer.js:355:14
    at execute (/var/www/html/ponchesme/backend/node_modules/telegraf/composer.js:370:13)
    at /var/www/html/ponchesme/backend/node_modules/telegraf/composer.js:355:14
    at execute (/var/www/html/ponchesme/backend/node_modules/telegraf/composer.js:370:13)
    at /var/www/html/ponchesme/backend/node_modules/telegraf/composer.js:355:14
    at execute (/var/www/html/ponchesme/backend/node_modules/telegraf/composer.js:370:13)
    at /var/www/html/ponchesme/backend/node_modules/telegraf/composer.js:355:14
    at execute (/var/www/html/ponchesme/backend/node_modules/telegraf/composer.js:370:13)
    at /var/www/html/ponchesme/backend/node_modules/telegraf/composer.js:355:14
    at execute (/var/www/html/ponchesme/backend/node_modules/telegraf/composer.js:370:13)
    at /var/www/html/ponchesme/backend/node_modules/telegraf/composer.js:355:14
    at execute (/var/www/html/ponchesme/backend/node_modules/telegraf/composer.js:370:13)
    at /var/www/html/ponchesme/backend/node_modules/telegraf/composer.js:355:14
    at execute (/var/www/html/ponchesme/backend/node_modules/telegraf/composer.js:370:13)
```

### Steps to Reproduce

I have absolutely no idea what's causing the error, hence I'm opening an issue looking forward to you helping me with a solution.

### Failure Logs

![bug](https://user-images.githubusercontent.com/44036750/92103051-193f5d00-ede8-11ea-8386-c544c8d7f956.jpg)

**Comment by palmerabollo** [comment]
may ba a duplicate of https://github.com/telegraf/telegraf/issues/1110

**Comment by KnorpelSenf** [comment]
Does this still happen when building from source? This might have been fixed by #1159

**Comment by wojpawlik** [comment]
If the issue persists, try reproducing it with low `--stack-trace-limit`.
